### PR TITLE
Detect and display pending TX found in mempool

### PR DESCRIPTION
**Pull request by LeeAlephium**
Requires new API, so based on update-schemas

### CODE REVIEW

**Comment by LeeAlephium** [comment]
Due to https://github.com/alephium/explorer-backend/issues/360, incoming pending won't work, but pending outgoing will, and eventually when that issue is resolved incoming should "just work".

**Review by nop33 — COMMENTED** [review]
Thanks for this PR Lee 🙏 

I've created #377 to address some of the issues I found in this PR.

**Comment by nop33** [comment]
I rebased since the base branch had diversed.

File: src/hooks/useTransactionInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    lockTime = lockTime.toISOString() === new Date(0).toISOString() ? undefined : lockTime
+  }
+
+  return {
+    amount,
+    direction,
+    infoType,
+    outputs,
+    lockTime,
+    label: {
```
Comment (mvaivre) [inline src/hooks/useTransactionInfo.tsx:78]: Mhh, I'm not sure that I like adding static data to a hook.
For me, hook's data should depend on a state. 

I'd create a new util method `getTransactionUI(type: 'in' | 'out' | 'move' | 'pending)` in  `utils/transactions` which return the needed labels, colors and components... WDYT?

Comment (nop33) [reply src/hooks/useTransactionInfo.tsx:78]: ok! Yeah, I also wansn't 100% satisfied, but I put it in the hook to reduce lines of code. Happy to move it out.

Comment (nop33) [reply src/hooks/useTransactionInfo.tsx:78]: It has to be a hook however, because it's using `t` and `useTheme`

Comment (mvaivre) [reply src/hooks/useTransactionInfo.tsx:78]: Okay yeah. Or we add `t` and `theme` as required arguments...

Comment (nop33) [reply src/hooks/useTransactionInfo.tsx]: Fixed!

Comment (nop33) [reply src/hooks/useTransactionInfo.tsx]: > Or we add t and theme as required arguments

I prefer the hook option, makes the code cleaner imo.

**Review by mvaivre — APPROVED** [review]

**Pull request merged**